erpnext/crm/doctype/contract/contract.json

```diff
@@ -50,7 +50,8 @@
    "fieldname": "is_signed",
    "fieldtype": "Check",
    "label": "Signed",
-   "no_copy": 1
+   "no_copy": 1,
+   "read_only": 1
```

Comment:
@DeeMysterio The `is_signed` field is used to toggle Signee Details Section. If it is made read only, how will you enter the signee details and how the `is_signed` field will be updated because currently there is no server side code to update it.
![Contract](https://user-images.githubusercontent.com/42651287/67973741-15c2ce80-fc37-11e9-9af2-3720f7325a2c.gif)